src/elli.erl

```diff
@@ -171,14 +176,14 @@ code_change(_OldVsn, State, _Extra) ->
 %%%===================================================================
 
 remove_acceptor(State, Pid) ->
-    State#state{acceptors = lists:delete(Pid, State#state.acceptors),
```

Comment:
Yes.

Florian, any suggestion on how an upgrade-friendly version would look like?

On Thu, Jun 11, 2015 at 3:47 PM, Andreas Hasselberg <
notifications@github.com> wrote:

> In src/elli.erl
> <https://github.com/knutin/elli/pull/90#discussion_r32221503>:
>
> > @@ -171,14 +176,14 @@ code_change(_OldVsn, State, _Extra) ->
> >  %%%===================================================================
> >
> >  remove_acceptor(State, Pid) ->
> > -    State#state{acceptors = lists:delete(Pid, State#state.acceptors),
>
> All acceptor processes are linked to this server, so they will also stop.
> Since the acceptor processes are used for handling the requests the active
> requests will also be stopped. I guess this isn't a big deal for most users
> of elli, but some might find it surprising.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/knutin/elli/pull/90/files#r32221503>.
>